lib/src/react_client/dart_interop_statics.dart

```diff
@@ -27,14 +28,15 @@ final ReactDartInteropStatics dartInteropStatics = (() {
           jsThis.setState(newObject());
         }
 
-        getRef(name) {
+        // ignore: omit_local_variable_types, prefer_function_declarations_over_variables
+        final RefMethod getRef = (name) {
```

Comment:
#nit could use a comment explaining why this isn't a function declaration

Reply:
Added in [60aec37](https://github.com/Workiva/react-dart/pull/366/commits/60aec372c4860242e91ab34d2c5ec2171ce54239)